core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -1294,6 +1294,19 @@ private static String toSql(RelNode root, SqlDialect dialect,
         .withPostgresql().ok(expectedPostgresql);
   }
 
+  @Test void testPercentileContWithinGroupClauseBigQuery() {
+    final String query = "select percentile_cont(0.5) WITHIN GROUP (ORDER BY `product_class_id`)\n"
+        + "from `foodmart`.`product`";
+    final String expected = "SELECT PERCENTILE_CONT(0.5) OVER "
```

Comment:
I think the PERCENTILE_CONT function for BigQuery must have two arguments. I think this expected query would probably fail since the column isn't passed to the function directly as an argument